### ISSUE TITLE
Change slides by .to() 

### ISSUE THREAD

**Issue by mahsaml90** [vitalii-andriiovskyi/ngx-owl-carousel-o#10]
(click)="owlCar.to('id')" doesn't change slide 
is there any other way?

```
<owl-carousel-o [options]="customOptions" #owlCar>
        <ng-container *ngFor="let image of product.images; let i = index">
            <ng-template carouselSlide [id]="product.id + i">
                <div class="gallery-item">
                    <a><img [src]="image" alt="Product"></a>
                </div>
            </ng-template>
        </ng-container>
    </owl-carousel-o> 
    <ul class="product-thumbnails">
        <li class="" *ngFor="let image of product.images | slice:0:5 ; let j = index">
            <a (click)="owlCar.to(product.id + j)">
                <img [src]="image" alt="Product.title"></a>
        </li>
    </ul>
```

**Comment by vitalii-andriiovskyi** [comment]
You wrote `<a (click)="owlCar.to(roduct.id + j)">`.

Maybe it should be `<a (click)="owlCar.to(product.id + j)">` (missing letter `p`). 
Method `owlCar.to('id')` works well. If it doesn't work, there's something wrong with `id`.

**Comment by mahsaml90** [comment]
i have misspelling in my comment
in my code i don't miss letter p


but i think the problem is that ` [id]="product.id + i"`  dose not set id to ng-template

**Comment by vitalii-andriiovskyi** [comment]
Did you set `nav: false` and `dots: false` in your options?

**Comment by mahsaml90** [comment]
yes both of them are false

**Comment by vitalii-andriiovskyi** [comment]
This is the reason why your custom navigation doesn't work. By default if the navigation is or gets disabled, methods `owlCar.to`, `owlCar.prev()`, and `owlCar.next()` stop working. 
This behavior of the carousel won't be changed. 

In you situation you can enable `dots` and make them invisible using CSS.

**Comment by mahsaml90** [comment]
If call `owlCar.to(id.toString())` from controller and set `nav: true` and `dots: true` it will wok well :)

but i don't want to have nav and dots in my owl
and also i think numeric id's must convert automatically

**Comment by mahsaml90** [comment]
Because of `||` i think both `nav `and `dots` must be true :(

**Comment by vitalii-andriiovskyi** [comment]
I removed that condition from the method `to(id)` and published new version v1.0.4. This is for Angular 7. 

Now the method will work in the case of disabling both `nav` and `dots`.

**Comment by mahsaml90** [comment]
It works well .
Thank you for your response.
